Documentation/artifact-format.md

```diff
@@ -95,6 +89,38 @@ Currently there is only version 1, but this document may describe later versions
 if they are created.
 
 
+manifest
+----
+
+Format: text
+
+Contains the below content exactly:
+
+```
+Version: 2
+Date: 2017-03-13 20:15:40.800231486 +0100 CET
+SHA256:
+ 1d0b820130ae028ce8a79b7e217fe505a765ac394718e795d454941487c53d32 0000/update.ext4
+ 4d480539cdb23a4aee6330ff80673a5af92b7793eb1c57c4694532f96383b619 header.tar.gz
+ 52c76ab66947278a897c2a6df8b4d77badfa343fec7ba3b2983c2ecbbb041a35 version
```

Comment:
Just out of curiosity, was this generated by some tool? I'm tempted to say we should stick to a standard format, which would mean to remove `SHA256:` and everything above it, remove the leading space, and have an extra space between the checksum and the file, like this:

```
1d0b820130ae028ce8a79b7e217fe505a765ac394718e795d454941487c53d32  0000/update.ext4
4d480539cdb23a4aee6330ff80673a5af92b7793eb1c57c4694532f96383b619  header.tar.gz
52c76ab66947278a897c2a6df8b4d77badfa343fec7ba3b2983c2ecbbb041a35  version
```

This matches the output of the `sha256sum` tool, which would be quite convenient. It's what we use for our license checking.